spec/garbage.dd

```diff
@@ -476,7 +476,7 @@ $(H2 $(LNAME2 precise_gc, Precise Heap Scanning))
 $(H2 $(LNAME2 precise_dataseg, Precise Scanning of the DATA and TLS segment))
 
     $(P $(B Windows only:) As of version 2.075, the DATA (global shared data)
-        and TLS segment (thread local data) of an executable
+        and [TLS](https://hosting.review/web-hosting-glossary/#12) segment (thread local data) of an executable
```

Comment:
`TLS` refers to [thread-local storage](https://en.wikipedia.org/wiki/Thread-local_storage) here, not Transport Layer Security.

Reply:
My mistake. Thanks.